pkg/cmd/hub/grpc_server.go

```diff
@@ -0,0 +1,27 @@
+package hub
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+	"k8s.io/utils/clock"
+
+	commonoptions "open-cluster-management.io/ocm/pkg/common/options"
+	"open-cluster-management.io/ocm/pkg/server/grpc"
+	"open-cluster-management.io/ocm/pkg/version"
+)
+
+func NewGRPCServerCommand() *cobra.Command {
+	opts := commonoptions.NewOptions()
+	grpcServerOpts := grpc.NewGRPCServerOptions()
+	cmdConfig := opts.NewControllerCommandConfig("grpc-server", version.Get(), grpcServerOpts.Run, clock.RealClock{})
```

Comment:
```suggestion
	cmdConfig := opts.NewControllerCommandConfig("grpc", version.Get(), grpcServerOpts.Run, clock.RealClock{})
```

Reply:
we need a component name here, `grpc-server` may be more suitable?

Reply:
en, I see could we customize the component name? grpc-server is not a good name given it will be used as part of lease lock